src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -416,7 +416,7 @@
 
   describe 'DELETE #destroy' do
     before do
-      login user_moi
+      login user
```

Comment:
Nevermind my previous comment. Though would you mind to squash this into the previous commit(s)?